src/seedu/addressbook/Main.java

```diff
@@ -40,10 +41,19 @@ public static void main(String... launchArgs) {
     /** Runs the program until termination.  */
     public void run(String[] launchArgs) {
         start(launchArgs);
+        login();
         runCommandLoopUntilExitCommand();
         exit();
     }
-
+    //Create a login function
+    public void login(){
+        while(true){
+            if(ui.Password().equals("password")) {
+                System.out.println("Welcome to the AddressBook!");
```

Comment:
"password" here acts like a magic string. Please refer to the coding style guide which mentions about magic strings and numbers. 

Also, check if this method "login" is violating SRP.